src/modules/translation/languages/en.json

```diff
@@ -894,9 +894,11 @@
   "curation_page": {
     "filter": {
       "all_status": "All status",
+      "not_reviewed": "Not reviewed",
       "approved": "Approved",
       "rejected": "Rejected"
     },
+    "search_placeholder": "Search between {count} results by name or owner address...",
```

Comment:
we can (marginally) improve this translation by adding plural like the key below!
`{count, plural, one {result} other {results}}`

we can skip it for zh